### PR TITLE
Add changes to support Micro

### PR DESCRIPTION
**Pull request by javmorcas**

### CODE REVIEW

File: build.bat [hunk 1]
```diff
@@ -11,11 +11,20 @@ set "java_folder=%script_location%srcJava"
 set "java_scripts_folder=%script_location%resource\java_scripts"
 set "bin_folder=%script_location%bin"
 
+@REM # By default we will build pro, not micro
+set BUILD_MICRO=0
+set BUILD_MICRO_24x_COMPATIBILITY=0
+<<<<<<< HEAD
```
Comment (alfonso-soto) [inline build.bat]: Fix this.

File: build.bat [hunk 2]
```diff
@@ -368,6 +430,62 @@ if %BUILD_JAVA% == 1 (
...
+	if not !ERRORLEVEL! == 0 (
+		echo [ERROR]: Failure compiling code for %classic_cpp_lang_string%.
+		cd ..
+		exit /b 1
+	)
+	cd ..
+
+	echo [INFO]: Copying perftest_cpp executable file:
+	md "%bin_folder%"\%architecture%\!RELEASE_DEBUG!
+	copy /Y "%classic_cpp_folder%"\objs\%architecture%\!RELEASE_DEBUG!\perftest_publisher.exe "%bin_folder%"\%architecture%\!RELEASE_DEBUG!\perftest_cpp_micro.exe
```
Comment (alfonso-soto) [inline build.bat]: At this point this path does not exist:
`"%classic_cpp_folder%"\objs\%architecture%\!RELEASE_DEBUG!\perftest_publisher.exe `

We are not creating the folder `!RELEASE_DEBUG!`
This should be `"%classic_cpp_folder%"\objs\%architecture%\perftest_publisher.exe`
Or create the folder `!RELEASE_DEBUG!`

**Comment by javmorcas** [comment]
Changes in #179

**Comment by javmorcas** [comment]
C

**Pull request closed without merging**